src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -40,6 +53,13 @@ private static Rotation getRotation(Block block) {
         return rotation;
     }
 
+    /**
+    * Give the new side of the block based on its' rotation
+    *
+    * @param block - the block that is about to be checked its' rotation and counted its' connections
+    * @param definedSides - the definedSides on which the loop iterates the sides of the block
```

Comment:
This one doesn't quite make sense. Try and re-write it without using `definedSides` in the description.